repo2docker/detectors.py

```diff
@@ -136,6 +139,16 @@ class BuildPack(LoggingConfigurable):
     and there are *some* general guarantees of ordering.
 
     """
+
+    jupyterhub_version = Unicode(
+        '0.7.2',
+        config=True,
+        help="""JupyterHub version to install.
+
+        In general, the JupyterHub version in the image
```

Comment:
Isn't this more than "in general"? I thought this was a hard requirement?

Reply:
That depends (and depends on the role of repo2docker to some degree). Right now, repo2docker can be used to build and run standalone images locally, in which case jupyterhub version doesn't matter.

Reply:
sounds good - just wanna make sure we're not leading people into the danger zone ;-)